<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Signup</class>
 <widget class="QDialog" name="Signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1203</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>900</width>
     <height>450</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>900</width>
     <height>450</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>900</width>
     <height>450</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="signup1">
    <widget class="QLineEdit" name="lineEdit_email">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>200</y>
       <width>300</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background-color: rgba(0, 0, 0, 0);
 	border: 2px solid black;
    border-bottom-color: #2CBB63;
	border-top-color: rgba(0, 0, 0, 0);
border-right-color: rgba(0, 0, 0, 0);
border-left-color: rgba(0, 0, 0, 0);

	
	color: black;
}


</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>ENTER EMAIL HERE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>310</y>
       <width>120</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
  border: none;
  border-radius: 8px;
  padding: 10px 20px;
  background-color: #2CBB63; /* Green color */
  color: white;
  font-size: 14px;
  font-weight: bold;
  text-transform: uppercase;
  transition: background-color 0.3s ease;
}

QPushButton:hover {
  background-color: #7AE59A; /* Lighter green color on hover */
}

QPushButton:pressed {
  background-color: #A2EFB6; /* Lighter green color when pressed */
  border: 5px solid #2CBB63; /* Green contour */
}

QPushButton:focus {
  outline: none;
  box-shadow: 0 0 0 3px rgba(0, 102, 204, 0.4);
}</string>
     </property>
     <property name="text">
      <string>START</string>
     </property>
    </widget>
    <widget class="QLabel" name="step1_1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>380</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">

QLabel {
  font-weight: bold;
  font-size: 16px;
color: #2CBB63;
}</string>
     </property>
     <property name="text">
      <string>STEP 1</string>
     </property>
    </widget>
    <widget class="QLabel" name="step2_1">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>380</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
QLabel {
  font-weight: bold;
  font-size: 16px;
color:black;
}</string>
     </property>
     <property name="text">
      <string>-----------STEP 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="step3_1">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>380</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
QLabel {
  font-weight: bold;
  font-size: 16px;
color:black;
}</string>
     </property>
     <property name="text">
      <string>-----------STEP 3</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>60</y>
       <width>371</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
  font-weight: bold; /* Texte en gras */
  font-size: 24px; /* Taille de police */
  color: black; /* Couleur du texte */
  background-color: transparent; /* Fond transparent */
}</string>
     </property>
     <property name="text">
      <string>YOUR MISSION START HERE</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>80</y>
       <width>281</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
  font-weight: bold; /* Texte en gras */
  font-size: 18px; /* Taille de police */
  color: #2CBB63; /* Couleur du texte */
  background-color: transparent; /* Fond transparent */
}</string>
     </property>
     <property name="text">
      <string>FIRST LET'S GET YOUR EMAIL.</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="signup2">
    <widget class="QLineEdit" name="lineEdit_username">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>260</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background-color: rgba(0, 0, 0, 0);
 	border: 2px solid black;
    border-bottom-color: #2CBB63;
	border-top-color: rgba(0, 0, 0, 0);
border-right-color: rgba(0, 0, 0, 0);
border-left-color: rgba(0, 0, 0, 0);

	
	color: black;
}

QLineEdit::placeholder {
    
	font: 87 10pt &quot;Arial&quot;;
}

</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>USERNAME</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_password">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>160</y>
       <width>260</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background-color: rgba(0, 0, 0, 0);
 	border: 2px solid black;
    border-bottom-color: #2CBB63;
	border-top-color: rgba(0, 0, 0, 0);
border-right-color: rgba(0, 0, 0, 0);
border-left-color: rgba(0, 0, 0, 0);

	
	color:black;
}

QLineEdit::placeholder {
    
	font: 87 10pt &quot;Arial&quot;;
}

</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>PASSWORD</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_confirmpass">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>240</y>
       <width>260</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background-color: rgba(0, 0, 0, 0);
 	border: 2px solid black;
    border-bottom-color: #2CBB63;
	border-top-color: rgba(0, 0, 0, 0);
border-right-color: rgba(0, 0, 0, 0);
border-left-color: rgba(0, 0, 0, 0);

	
	color: black;
}

QLineEdit::placeholder {
    
	font: 87 10pt &quot;Arial&quot;;
}

</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>CONFIRM PASSWORD</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_pass_err1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>190</y>
       <width>311</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>Password must be at least 8 characters long.</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_pass_err2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>210</y>
       <width>311</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>Password must contain at least one letter and one non-letter character.</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_pass_err3">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>280</y>
       <width>311</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>Does not match password field.</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_next">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>310</y>
       <width>120</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
  border: none;
  border-radius: 8px;
  padding: 10px 20px;
  background-color: #2CBB63; /* Green color */
  color: white;
  font-size: 14px;
  font-weight: bold;
  text-transform: uppercase;
  transition: background-color 0.3s ease;
}

QPushButton:hover {
  background-color: #7AE59A; /* Lighter green color on hover */
}

QPushButton:pressed {
  background-color: #A2EFB6; /* Lighter green color when pressed */
  border: 5px solid #2CBB63; /* Green contour */
}

QPushButton:focus {
  outline: none;
  box-shadow: 0 0 0 3px rgba(0, 102, 204, 0.4);
}</string>
     </property>
     <property name="text">
      <string>NEXT</string>
     </property>
    </widget>
    <widget class="QLabel" name="step1_4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>380</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
QLabel {
  font-weight: bold;
  font-size: 16px;
color:black;
}</string>
     </property>
     <property name="text">
      <string>STEP 1------------------------------</string>
     </property>
    </widget>
    <widget class="QLabel" name="step1_5">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>380</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">

QLabel {
  font-weight: bold;
  font-size: 16px;
color: #2CBB63;
}</string>
     </property>
     <property name="text">
      <string>STEP 2</string>
     </property>
    </widget>
    <widget class="QLabel" name="step1_6">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>380</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
QLabel {
  font-weight: bold;
  font-size: 16px;
color:black;
}</string>
     </property>
     <property name="text">
      <string>-----------STEP 3</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
  font-weight: bold; /* Texte en gras */
  font-size: 24px; /* Taille de police */
  color: black; /* Couleur du texte */
  background-color: transparent; /* Fond transparent */
}</string>
     </property>
     <property name="text">
      <string>CREATE A USERNAME AND</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
  font-weight: bold; /* Texte en gras */
  font-size: 24px; /* Taille de police */
  color: black; /* Couleur du texte */
  background-color: transparent; /* Fond transparent */
}</string>
     </property>
     <property name="text">
      <string>PASSWORD</string>
     </property>
    </widget>
    <zorder>label_pass_err1</zorder>
    <zorder>label_pass_err2</zorder>
    <zorder>label_pass_err3</zorder>
    <zorder>lineEdit_confirmpass</zorder>
    <zorder>lineEdit_username</zorder>
    <zorder>lineEdit_password</zorder>
    <zorder>pushButton_next</zorder>
    <zorder>step1_4</zorder>
    <zorder>step1_5</zorder>
    <zorder>step1_6</zorder>
    <zorder>label_7</zorder>
    <zorder>label_8</zorder>
   </widget>
   <widget class="QWidget" name="signup3">
    <widget class="QPushButton" name="pushButton_signup">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>310</y>
       <width>120</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
  border: none;
  border-radius: 8px;
  padding: 10px 20px;
  background-color: #2CBB63; /* Green color */
  color: white;
  font-size: 14px;
  font-weight: bold;
  text-transform: uppercase;
  transition: background-color 0.3s ease;
}

QPushButton:hover {
  background-color: #7AE59A; /* Lighter green color on hover */
}

QPushButton:pressed {
  background-color: #A2EFB6; /* Lighter green color when pressed */
  border: 5px solid #2CBB63; /* Green contour */
}

QPushButton:focus {
  outline: none;
  box-shadow: 0 0 0 3px rgba(0, 102, 204, 0.4);
}</string>
     </property>
     <property name="text">
      <string>SIGN UP</string>
     </property>
    </widget>
    <widget class="QLabel" name="step1_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>380</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
QLabel {
  font-weight: bold;
  font-size: 16px;
color:black;
}</string>
     </property>
     <property name="text">
      <string>STEP 1-----------------</string>
     </property>
    </widget>
    <widget class="QLabel" name="step2_3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>380</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
QLabel {
  font-weight: bold;
  font-size: 16px;
color:black;
}</string>
     </property>
     <property name="text">
      <string>STEP 2--------------------</string>
     </property>
    </widget>
    <widget class="QLabel" name="step3_3">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>380</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">

QLabel {
  font-weight: bold;
  font-size: 16px;
color: #2CBB63;
}</string>
     </property>
     <property name="text">
      <string>STEP 3</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nom">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>130</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background-color: rgba(0, 0, 0, 0);
    border: 2px solid black;
    border-bottom-color: #2CBB63;
    border-top-color: rgba(0, 0, 0, 0);
    border-right-color: rgba(0, 0, 0, 0);
    border-left-color: rgba(0, 0, 0, 0);
    color: black;
}

QLineEdit::placeholder {
    color: black;
}

QLineEdit::!enabled::placeholder {
    color: rgba(0, 0, 0, 127);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>NAME</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="date_naissance">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>210</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>50</weight>
       <bold>false</bold>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QDateEdit {
  padding: 0;
  border: none; /* No border */
  background-color: transparent; /* Transparent background */
  color: black; /* Black text color */
  border-bottom: 2px solid #808080; /* Grey underline */
}

QDateEdit:hover {
  border-bottom: 2px solid #2CBB63; /* Green underline on hover */
  font-weight: bold; /* Bolder text */
  padding-bottom: 2px; /* Increase bottom padding */
}</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="accelerated">
      <bool>false</bool>
     </property>
     <property name="date">
      <date>
       <year>1998</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_prenom">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>110</y>
       <width>130</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background-color: rgba(0, 0, 0, 0);
 	border: 2px solid black;
    border-bottom-color: #2CBB63;
	border-top-color: rgba(0, 0, 0, 0);
border-right-color: rgba(0, 0, 0, 0);
border-left-color: rgba(0, 0, 0, 0);

	
	color:black;
}

QLineEdit::placeholder {
    
	font: 87 10pt &quot;Arial&quot;;
}

</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>LAST-NAME</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_cin">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>130</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background-color: rgba(0, 0, 0, 0);
 	border: 2px solid black;
    border-bottom-color: #2CBB63;
	border-top-color: rgba(0, 0, 0, 0);
border-right-color: rgba(0, 0, 0, 0);
border-left-color: rgba(0, 0, 0, 0);

	
	color: black;
}

QLineEdit::placeholder {
    
	font: 87 10pt &quot;Arial&quot;;
}

</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>CIN</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_tel">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>160</y>
       <width>130</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background-color: rgba(0, 0, 0, 0);
 	border: 2px solid black;
    border-bottom-color: #2CBB63;
	border-top-color: rgba(0, 0, 0, 0);
border-right-color: rgba(0, 0, 0, 0);
border-left-color: rgba(0, 0, 0, 0);

	
	color: black;
}

QLineEdit::placeholder {
    
	font: 87 10pt &quot;Arial&quot;;
}

</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>PHONE-NUMBER</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_sexe">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>220</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
  padding: 0;
  border: none; /* No border */
  background-color: transparent; /* Transparent background */
  color: black; /* Black text color */
  border-bottom: 2px solid #808080; /* Grey underline */
}

QComboBox:hover {
  border-bottom: 4px solid #2CBB63; /* Green underline on hover */
  font-weight: bold; /* Bolder text */
  padding-bottom: 2px; /* Increase bottom padding */
}</string>
     </property>
     <item>
      <property name="text">
       <string>Homme</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Femme</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBox_status">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>260</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox {
  padding: 0;
  border: none; /* No border */
  background-color: transparent; /* Transparent background */
  color: black; /* Black text color */
  border-bottom: 2px solid #808080; /* Grey underline */
}

QComboBox:hover {
  border-bottom: 4px solid #2CBB63; /* Green underline on hover */
  font-weight: bold; /* Bolder text */
  padding-bottom: 2px; /* Increase bottom padding */
}</string>
     </property>
     <item>
      <property name="text">
       <string>Celibataire</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>marie</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>divorce</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>351</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
  font-weight: bold; /* Texte en gras */
  font-size: 24px; /* Taille de police */
  color: black; /* Couleur du texte */
  background-color: transparent; /* Fond transparent */
}</string>
     </property>
     <property name="text">
      <string>COMPLETE YOUR PERSONAL</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>231</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
  font-weight: bold; /* Texte en gras */
  font-size: 24px; /* Taille de police */
  color: black; /* Couleur du texte */
  background-color: transparent; /* Fond transparent */
}</string>
     </property>
     <property name="text">
      <string> INFORMATION</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>350</y>
       <width>47</width>
       <height>14</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>1271</width>
     <height>611</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="image.qrc">:/img/d9c105d19a69658c8c819ac633d9eb64.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="image.qrc">:/img/alleco.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>stackedWidget</zorder>
  <zorder>label_5</zorder>
 </widget>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
